public/app/features/commandPalette/KBarResults.tsx

```diff
@@ -0,0 +1,226 @@
+import { ActionImpl, getListboxItemId, KBAR_LISTBOX, useKBar } from 'kbar';
+import { usePointerMovedSinceMount } from 'kbar/lib/utils';
+import * as React from 'react';
+import { useVirtual } from 'react-virtual';
+
+// From https://github.com/timc1/kbar/blob/main/src/KBarResults.tsx
+// TODO: Go back to KBarResults from kbar when https://github.com/timc1/kbar/issues/2815 is fixed
+// Remember to remove dependency on react-virtual when removing this file
+
+const START_INDEX = 0;
+
+interface RenderParams<T = ActionImpl | string> {
+  item: T;
+  active: boolean;
+}
+
+interface KBarResultsProps {
+  // eslint-disable-next-line
```

Comment:
think we should be explicit and only disable the rule we care about here
```suggestion
  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```